project/Common.scala

```diff
@@ -39,6 +39,23 @@ object Common extends AutoPlugin {
       "-Xfuture",
       "-target:jvm-1.8"
     ),
+    Compile / doc / scalacOptions := scalacOptions.value ++ Seq(
+      "-doc-title",
+      "Alpakka",
+      "-doc-version",
+      version.value,
+      "-sourcepath",
+      (baseDirectory in ThisBuild).value.toString,
+      "-doc-source-url", {
+        val branch = if (isSnapshot.value) "master" else s"v$version"
+        s"https://github.com/akka/alpakka/tree/${branch}$${sourcepath.name}.scala#L1"
```

Comment:
This doesn't work for Java sources, we should leave it until https://github.com/scala/scala/pull/7532 becomes available with Scaladoc 2.13.

Reply:
Or do the post-processing as suggested in 2012(!): https://github.com/scala/bug/issues/5388#issuecomment-292401894